### ISSUE TITLE
Error when running the cifar10_gen script

### ISSUE THREAD

**Issue by Sentient07** [kundan2510/pixelCNN#2]
Hi, 
I was trying to benchmark the results of PixelCNN on CIFAR10 and I when I ran the script, I landed into this [error](http://dpaste.com/33R4231). I am using Theano 0.9.0 and Lasagne 0.2.dev. I would be glad if you could help me with this error, or share the results of the CIFAR10(the weights and generated images). 

Thanks, 
Ramana

**Comment by Sentient07** [comment]
The main concern is that when I'm using a similar architecture, the gradients that I observe while training are very low and in the range of 1e-4. Hence, I wanted to check with your implementation the values of gradients.

**Comment by kundan2510** [comment]
Sorry for late response. I never checked the values of the gradients, however training was super slow for me when using 256-way softmax. You may look at pixelCNN++ where they modify the cost and architecture a bit to get speed up in training.

**Comment by Sentient07** [comment]
Hi Kundan, thanks for the points. We have proceeded with the pixelCNN++ itself later. Also, for using the 256 way softmax, I think PixelVAE gave very good results(for someone reading the thread in the future). For the error I reported, it's fixed in Theano. 

Cheers, 
Ramana
